models/points.go

```diff
@@ -678,6 +678,13 @@ func scanTags(buf []byte, i int, indices []int) (int, int, []int, error) {
 		case tagValueState:
 			state, i, err = scanTagsValue(buf, i)
 		case fieldsState:
+			// Grow our indices slice if needed
+			if commas >= len(indices) {
+				// this should be the last tag
```

Comment:
Same here, I wasn't sure why this comment would be true. Suggested clarification:
```suggestion
				// The parser is in `fieldsState`, so there are no more tags. We only need 1 more entry in the slice to store the final entry.
```